spec/requests/api/v1/participants_spec.rb

```diff
@@ -23,13 +23,15 @@
              school_cohort: school_cohort)
     end
     let!(:withdrawn_ect_profile_record) { create(:participant_profile, :withdrawn_record, :ect, school_cohort: school_cohort) }
+    let(:user) { create(:user) }
+    let(:early_career_teacher_profile) { create(:participant_profile, :ect, school_cohort: school_cohort, user: user) }
```

Comment:
We could put those lets in the `describe` that is using them. Also, do we need to specify `user`?